backend.py

```diff
@@ -4,6 +4,7 @@
 from pathlib import Path
 from collections import OrderedDict
 from random import shuffle
+import yaml
```

Comment:
`yaml`, nebo `pyyaml`? V testech mám pyyaml. Jedná se o stejnou knihovnu, která jde naimportovat různými způsoby, nebo dvě různé?

Reply:
V testech taky *importuješ* `yaml`. Je to jedna ze starých, nekonzistentně pojmenovaných knihoven: [instaluje se](https://github.com/PyLadiesCZ/roboprojekt/blob/master/requirements.txt#L4) jako [`PyYAML`](https://pypi.org/project/PyYAML/) ale importuje jako `yaml`.

Reply:
Dik, jsem dnes uplne mimo!